File: test/Data.Analyzers.Vsix.Test/VisualBasic/InvalidRegisterMounterGetterParam.vb
Public Class InvalidRegisterMounterGetterParam
    Inherits Model

    Private Class AnotherModel
        Inherits Model

        Private m_Column1 As _Int32
        Public Property Column1 As _Int32
            Get
                Return m_Column1
            End Get
            Private Set
                m_Column1 = Value
            End Set
        End Property
    End Class

    Shared Sub New()
        RegisterColumn(Function(x As InvalidRegisterMounterGetterParam) x.Column1)
        RegisterColumn(Function(x As AnotherModel) x.Column1)
    End Sub

    Private m_Column1 As _Int32
    Public Property Column1 As _Int32
        Get
            Return m_Column1
        End Get
        Private Set
            m_Column1 = Value
        End Set
    End Property
End Class

File: test/Data.Analyzers.Vsix.Test/CSharp/InvalidRegisterMounterGetterParam.cs
namespace DevZest.Data.Analyzers.Vsix.Test.CSharp
{
    public class InvalidRegisterMounterGetterParam : Model
    {
        private class AnotherModel : Model
        {
            public _Int32 Column1 { get; private set; }
        }

        static InvalidRegisterMounterGetterParam()
        {
            RegisterColumn((InvalidRegisterMounterGetterParam _) => _.Column1);
            RegisterColumn((AnotherModel _) => _.Column1);
        }

        public _Int32 Column1 { get; }
    }
}
